deps/wazuh_testing/wazuh_testing/migration_tool/utils.py

```diff
@@ -0,0 +1,178 @@
+    '''Select the newest delta file generated (where the results are) from the list of all files.
+
+    Args:
+        deltas_filepath (str): Path where the files are located.
+    Returns:
+        newest_file (str): Path of the newest file.
+    '''
+    all_files = glob.glob(os.path.join(deltas_filepath, '*.delta.*'))
+    newest_file = max(all_files, key=os.path.getctime)
+
+    return newest_file
+
+
+def sanitize_configuration(configuration):
+    '''Normalize the configuration for it to be correctly processed and compatible.
+
+    Args:
+        configuration (list): Test case configuration to be sanitized.
+    Returns:
+        configuration (list): Configuration normalized.
+    '''
+    for configurations_obj in configuration:
+        configurations_list = configurations_obj['configurations']
+        for config_obj in configurations_list:
+            for key in config_obj:
+                config_obj[key.lower()] = config_obj.pop(key)
+
+    return configuration
+
+
+def validate_json_against_schema(json_document, schema):
+    '''Validate a JSON document under the given schema.
+    Args:
+        json_document (str): Path of the JSON document to be validated
+        schema (str): Path of the CVE5 Schema by default.
+    Returns:
+        result (bool): False if the validation thrown an error, True if not.
+        error (str): Error in the JSON document.
+    '''
+    schema = read_json_file(schema)
+
+    try:
+        validate(instance=json_document, schema=schema)
+    except ValidationError as err:
+        return False, err.message
+
+    return True, ''
+
+
+def validate_against_delta_schema(_elements):
+    '''Wrapper function
+    '''
+    _result = True
+    _errors = []
+    for cve in _elements:
+        _result, _error = validate_json_against_schema(cve, DELTA_SCHEMA_PATH)
+        if _result is False:
+            _errors.append(_error)
+
+    return _errors
+
+
+def validate_against_cve5_schema(_elements):
+    '''Wrapper function
+    '''
+    _result = True
+    _errors = []
+
+    for cves in _elements:
+        data = json.loads(cves['data_blob'])
+        _result, _error = validate_json_against_schema(data, CVE5_SCHEMA_PATH)
+        if _result is False:
+            _errors.append(_error)
+
+    return _errors
+
+
+def query_publisher_db(query):
+    '''Function to query the DB created by Content Migration Tool.
+    Args:
+        query (str): Query to send to the DB.
+    Returns:
+        result (list): Query results, empty if no query was executed or no results were returned.
+    '''
+    result = []
+
+    try:
+        cnx = mysql.connector.connect(
+            host="localhost",
+            user="test",
+            password="Test123$",
+            database="test_database"
+            )
+    except mysql.connector.Error as err:
+        cnx = None
+        if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+            logger.error("Something is wrong with your user name or password")
```

Comment:
```suggestion
            logger.error('Something is wrong with your user name or password')
```

Apply this to all ocurrences

Reply:
Done in b29961769